phpstan.neon

```diff
@@ -1,7 +1,7 @@
 parameters:
-    level: 4
+    level: 5
     paths:
         -   Context
         -   api
         -   contrib
-        -   sdk
\ No newline at end of file
+        -   sdk
```

Comment:
Newline, meaning the project is missing `.editorconfig`

Reply:
Should we add a `.editorconfig`?

Reply:
Don't see why not, it can only help with the consistency of the changes.

Reply:
@dkarlovi - do you have a particular preference on what should be included in our `.editorconfig`?

Reply:
We can start with something very simple and tweak when needed.

Example
https://github.com/flexolabs/reddit-client/blob/master/.editorconfig

Reply:
Sounds good to me.

Reply:
@bobstrecansky added in https://github.com/open-telemetry/opentelemetry-php/pull/207